config.toml

```diff
@@ -85,7 +85,7 @@ HoneycombAPI = "https://api.honeycomb.io"
 # trace. This timer is in seconds. Set to 0 for immediate
```

Comment:
The comment reads `This timer is in seconds.` Does that need to be updated to state this is a duration? Does setting to 0 for immediate sends still work?

Reply:
ah good catch, I'll update the comment. 0 does work as expected